test/test_mps.py

```diff
@@ -6942,6 +6942,29 @@ def helper(shape, diag=0):
         helper((2, 8, 4, 5), diag=-2)
         helper((2, 8, 4, 5), diag=-3)
 
+    def test_lu_factor(self):
+        def helper(shape):
```

Comment:
This doesn't test the `pivot` kwarg.

Reply:
Thanks for pointing it out! I planned to remove this test and rely on the OpInfo test only after the unimplemented backward pass is completed. I will add a TODO here.

Reply:
Then, rather than adding this test, enable the OpInfo for this operation and xfail the gradient tests on mps.

Reply:
Done.